bin/create.js

```diff
@@ -219,84 +227,70 @@ function handleCreate (argv) {
                 argv.name = md5.update(fol, 'utf8').digest('hex');
             }
         }
-    }
-    else if (argv.name && argv.output !== 'none') {
-        throw new Error('The `name` option can only be specified when --output is set to `url`.');
+    } else if (argv.name && argv.output !== 'none') {
+        // throw Cli.usageError('The `name` option can only be specified when --output is set to `url`.');
```

Comment:
The check was introduced here: https://github.com/auth0/wt-cli/commit/8225af33eda4339e6889cbcf9298dcbdf29fbf1f#diff-681b7ffb8e95b8fcf806315ed20806ccR175

I need a hidden `output='none'` mode for the cron command since that re-uses the create logic but wants to opt out of its output.